src/DocumentFormat.OpenXml/Packaging/OpenXmlPart.cs

```diff
@@ -76,19 +76,19 @@ internal void Load(OpenXmlPackage? openXmlPackage, OpenXmlPart? parent, Uri uriT
             // TODO: should we delay load?
             var metroPart = openXmlPackage.Package.GetPart(uriTarget);
 
-            if (IsContentTypeFixed && metroPart.ContentType != ContentType)
+            _packagePart = metroPart;
+
+            if (!IsValidContentType(metroPart))
```

Comment:
I think you'll want to keep the `IsContentTypeFixed` check here before calling out to the method

Reply:
yes